proxy/proxy.go

```diff
@@ -209,6 +213,10 @@ func (p *T) AsyncProduce(topic string, key, message sarama.Encoder) {
 // available for consumption. In that case the user should back off a bit
 // and then repeat the request.
 func (p *T) Consume(group, topic string, ack Ack) (consumer.Message, error) {
+	if p.cfg.Consumer.Disabled {
+		return consumer.Message{}, ErrUnavailable
```

Comment:
I would rather make it `ErrDisabled`. Any time I see `something or something` error that makes me cry in frustration: "Why are you uncertain, why?!" :-)

Reply:
Hehe, yes very fair :)

Useful error messages is one of the things we try to focus on at Stripe, so shame on me. Pushed a fix.